app/index.html

```diff
@@ -411,6 +432,19 @@ <h2>Find your next TV Series!</h2>
         const minAudience = $("#audience_score").val();
         const genres = $("#genre").val();
         const networks = $("#network").val();
+
+        localStorage.setItem(
+          'initialFilters',
+          JSON.stringify({
+            minYear,
+            maxYear,
+            minTomatoMeter,
+            minAudience,
+            genres,
+            networks
```

Comment:
could we add ordering in here?

Reply:
Order the keys, or order the value array?

Reply:
i mean the adding the ordering of the data in the table :)

Reply:
Oh! Yes, I can do that. 

As for storing this in the URL, I can see about encoding this stuff with query params so the values are easily visible when sharing.

Reply:
that would be cool too!

Reply:
any updates on this?